install.sh

```diff
@@ -1,6 +1,70 @@
 #!/bin/sh
 
 # Overridden variables from Go template: {{.Overrides}}
+podman_install() {
+    [ -n "${KEEPALIVE_INTERVAL}" ] && ARGS="$ARGS -e SHELLHUB_KEEPALIVE_INTERVAL=$KEEPALIVE_INTERVAL"
+    [ -n "${PREFERRED_HOSTNAME}" ] && ARGS="$ARGS -e SHELLHUB_PREFERRED_HOSTNAME=$PREFERRED_HOSTNAME"
+    [ -n "${PREFERRED_IDENTITY}" ] && ARGS="$ARGS -e SHELLHUB_PREFERRED_IDENTITY=$PREFERRED_IDENTITY"
+
+    echo "📥 Downloading ShellHub container image..."
+
+    {
+        $SUDO podman pull -q shellhubio/agent:$AGENT_VERSION
```

Comment:
Hi, @No9 

That is amazing, but I found a small problem in this line.

If the user doesn't have the `docker.io` defined in the registries, this command will fail like this:

```
Error: short-name "shellhubio/agent:v0.18.3" did not resolve to an alias and no unqualified-search registries are defined in "/etc/containers/registries.conf"
```

Could you, please, modify it to something like this?

```suggestion
        $SUDO podman pull -q docker.io/shellhubio/agent:$AGENT_VERSION
```

Reply:
Updated in follow up commit